src/libPMacc/PMaccConfig.cmake

```diff
@@ -29,7 +29,7 @@
 ###############################################################################
 # PMacc
 ###############################################################################
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
The version used was pretty arbitrary. No features were used that require such a high version and PIConGPU runs also with less. Reason for changing this was that the most common cmakes are not that new (Cluster installations, linux package managers)
No need to add my name because this is a (too) minor change to claim authorship